pkg/utils/erofs/erofs.go

```diff
@@ -12,12 +12,22 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-func Mount(bootstrapPath, fsID, mountPoint string) error {
+func Mount(bootstrapPath, domainID, fscacheID, mountPoint string) error {
 	mount := unix.Mount
+	var opts string
 
-	opts := "fsid=" + fsID
+	if domainID != "" {
```

Comment:
It seems always `domainID != ""`.

Reply:
It can be loaded from rafs configuration json file